script.js

```diff
@@ -0,0 +1,116 @@
+
+
+  var country = prompt("What country do you desire to rule? You may choose from: China (c), Russia (r), North Korea(nk), USA(usa) or Canada(cn).").toLowerCase();
+  if (country === "c") {
+    alert("Congratulations on choosing China! You must be feeling peaceful today. Next, you will decide to learn industry or monastary.");
+  } else if (country === "r") {
+    alert("Congratulations on choosing Russia! You must be full of yourself today. Next, you will decide how you would like to move forward from the missle silo.");
+  } else if (country === "nk") {
+    alert("Congratulations on choosing North Korea! You must be feeling mysterious today. Next, you will decide how to over-throw the world.");
+  } else if (country === "usa") {
+    alert("Congratulations on choosing USA! You must be happy today. Next, changing the world one empire at a time.");
+  } else if (country === "cn") {
+    alert("Congratulations on choosing Canada! You must be angry today. But the world needs their maple syrup!");
+  };
+  do {
+    var name = prompt("Wait! What was your name?");
+    } while (!name);
+    console.log(name);
+
+
+  var style = prompt("Choose wisely, for your first action in " + country + "  you excel at: farm, industry or diplomacy.").toLowerCase();
+  if (style === "farm") {
+    alert("You have not chosen wisely. Youre country did not industrilize so, Game over!");
+
+  } else if (style === "industry") {
+    alert("Splendid choice! You have started building a city. Are you up to code on radiation building.");
+  } else if (style === "diplomacy") {
+    alert("Interesting choice! You have started diplomatic relations. Choose your peoples future.");
+  };
+
+
+  var fate = prompt("Your next choice decide's your peoples fate. Choose wisley. Choose: 1, 2, 3 or 4.");
+  if (fate === "1") {
+    alert("You have not chosen wisely. Alongside the diplomatic relations you became a thief and were charged at the International Criminal Court; thrown in jail. Game over! (Click OK to see your score!)");
+  }
+  else if (fate === "2") {
+    alert("Splendid choice, again! You have chosen the way of the farm. You have become a monk and will live for all of eternity. You win! (Click OK to see your score!)");
+  }
+  else if (fate === "3") {
+    alert("You have become to powerful. Your city has leeped into chaos with the pending onslaught of the nuclear missles. You have become a knight and ruled your city poorly. The citizens have overpowered you and sent you to the guillotine. Game over! (Click OK to see your score!)");
+  }
+  else if (fate === "4") {
+    alert("Splendid choice, again! You have shown the world you are a true leader. Because of your Communist ways you have used your nuclear power and destroyed your enemies and farm land.  While you have won you are now alone an empitent. You win! (Click OK to see your score!)");
+  }
+
+/* below still needs corrections based on objects and running syntex errors.
+  var p1, p2, p3, p4;
+
```

Comment:
It's fine to comment out code briefly to debug or test it out, but make sure to clean it up after and remove these additional functions, etc that are not being used